app/src/main/java/com/gdfp/android_evaluationv2/TaskDetailActivity.java

```diff
@@ -134,7 +130,7 @@ private void getData() {
         }
 
         // Set the description of the Task on the textDate TextView
-        textDescription.setText(taskDescription);
+        textDescription.setText();
```

Comment:
I think the problem is in the TaskTitleView  class you need to review this class